NicolaeChristof/Rise
# Feature end game

Implemented an end game state for when the player reaches the top of the tree.
Created a second track system for the end game state.

To test:
Climb to the top of the tree. Once you reach the top the player movement should stop and the camera view will switch to the end game camera which circles around the tree top.
